### PR TITLE
[#2253] Prevent project title from expanding too far

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2253-limit-width-of-project-name-container

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/aa7f21165597bf3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/aa7f21165597bf3/output.txt

Total script time: 0.97 mins

Server is running on http://pr1003.butyr.org/templates/basic/
Run the Test Suite: http://pr1003.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/244e51f0dab995e/output.txt

File: css/header.less [hunk 1]
```diff
@@ -106,12 +106,24 @@
...
+  }
   .icon {
     opacity: 0.4;
-    position: relative;
+    position: absolute;
     visibility: hidden;
-    left: 10px;
-    top: 2px;
+    right: 0;
+    top: 22px;
```
Comment (k88hudson) [inline css/header.less]: Better to do this as a fraction of `@HEADER_SPACING`

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/244e51f0dab995e/output.txt

Total script time: 0.96 mins

Server is running on http://pr1003.butyr.org/templates/basic/
Run the Test Suite: http://pr1003.butyr.org/test/

File: css/header.less [hunk 2]
```diff
@@ -106,12 +106,24 @@
...
+  }
   .icon {
     opacity: 0.4;
-    position: relative;
+    position: absolute;
     visibility: hidden;
-    left: 10px;
-    top: 2px;
+    right: 0;
+    top: @HEADER_SPACING / 2.7;
```
Comment (secretrobotron) [inline css/header.less]: What is 2.7? Please add an explanation and make this a const.

File: css/header.less [hunk 3]
```diff
@@ -106,12 +109,26 @@
...
 .butter-project-title {
+  max-width: 35%;
+  min-width: 8%;
+  height: 100%;
+
+  a {
+    &.butter-project-name {
+      display: block;
+      text-align: left;
+      width: 100%;
```
Comment (secretrobotron) [inline css/header.less]: I don't think you need this property, or the `text-align` one. `white-space: nowrap` seems to have been the key.

**Pull request merged**